AdsAdManager/README.md

```diff
@@ -0,0 +1,45 @@
+# Google Ads Ad Manager for PHP
+
+> Idiomatic PHP client for [Google Ads Ad Manager](https://developers.google.com/ad-manager/api/beta).
```

Comment:
The first link here should be a link to the product landing page (as opposed to the API documentation):
```suggestion
> Idiomatic PHP client for [Google Ads Ad Manager](https://developers.google.com/ad-manager).
```